src/components/ChallengeEditor/ChallengeReviewer-Field/index.js

```diff
@@ -323,44 +453,46 @@ class ChallengeReviewerField extends Component {
           </div>
         )}
 
-        <div className={styles.formRow}>
-          <div className={styles.formGroup}>
-            <label>Review Type:</label>
-            {readOnly ? (
-              <span>
-                {(() => {
-                  const typeMap = {
-                    [REVIEW_OPPORTUNITY_TYPES.REGULAR_REVIEW]: 'Regular Review',
-                    [REVIEW_OPPORTUNITY_TYPES.COMPONENT_DEV_REVIEW]: 'Component Dev Review',
-                    [REVIEW_OPPORTUNITY_TYPES.SPEC_REVIEW]: 'Spec Review',
-                    [REVIEW_OPPORTUNITY_TYPES.ITERATIVE_REVIEW]: 'Iterative Review',
-                    [REVIEW_OPPORTUNITY_TYPES.SCENARIOS_REVIEW]: 'Scenarios Review'
-                  }
-                  return typeMap[reviewer.type] || 'Regular Review'
-                })()}
-              </span>
-            ) : (
-              <select
-                value={reviewer.type || REVIEW_OPPORTUNITY_TYPES.REGULAR_REVIEW}
-                onChange={(e) => this.updateReviewer(index, 'type', e.target.value)}
-              >
-                <option value={REVIEW_OPPORTUNITY_TYPES.REGULAR_REVIEW}>Regular Review</option>
-                <option value={REVIEW_OPPORTUNITY_TYPES.COMPONENT_DEV_REVIEW}>Component Dev Review</option>
-                <option value={REVIEW_OPPORTUNITY_TYPES.SPEC_REVIEW}>Spec Review</option>
-                <option value={REVIEW_OPPORTUNITY_TYPES.ITERATIVE_REVIEW}>Iterative Review</option>
-                <option value={REVIEW_OPPORTUNITY_TYPES.SCENARIOS_REVIEW}>Scenarios Review</option>
-              </select>
-            )}
+        {!this.isAIReviewer(reviewer) && (
+          <div className={styles.formRow}>
+            <div className={styles.formGroup}>
+              <label>Review Type:</label>
+              {readOnly ? (
+                <span>
+                  {(() => {
+                    const typeMap = {
```

Comment:
this map either already exists somewhere, or you have to create it in a config/constants file. this is bad practice (to create it in the render function, it gets recreated a log of times).